Dockerfile

```
@@ -33,8 +33,7 @@ LABEL org.opencontainers.image.source="https://github.com/IgnisDa/ryot"
 LABEL org.opencontainers.image.description="The only self hosted tracker you will ever need!"
 ENV FRONTEND_UMAMI_SCRIPT_URL="https://umami.diptesh.me/script.js"
 ENV FRONTEND_UMAMI_WEBSITE_ID="5ecd6915-d542-4fda-aa5f-70f09f04e2e0"
-RUN apt-get update && apt-get install -y --no-install-recommends wget curl ca-certificates procps libc6
-RUN wget http://ftp.debian.org/debian/pool/main/o/openssl/libssl1.1_1.1.1w-0+deb11u1_${TARGETARCH}.deb && dpkg -i libssl1.1_1.1.1w-0+deb11u1_${TARGETARCH}.deb && rm -rf libssl1.1_1.1.1w-0+deb11u1_${TARGETARCH}.deb
+RUN apt-get update && apt-get install -y --no-install-recommends wget curl ca-certificates procps libc6 libssl
```

Hadolint (2.12.0)

[info] Delete the apt-get lists after installing something

(DL3009)